examples/jira/jira_get_comment.py

```diff
@@ -0,0 +1,24 @@
+""" Example: Get Comments on a Jira Issue
+"""
+# coding=utf-8
+from atlassian import Jira
+
+
+def main():
+    jira = Jira(url="https://atlassian-python.atlassian.net/", username="martyn.bristow@gmail.com", password="...")
+
+    print("\n*Get Comments on INT-1*")
+    comments = jira.issue_get_comments("INT-1")
+    print(comments)
+
+    print("\n*Get a single comment on INT-1*")
+    comments = jira.issue_get_comment(10000, 10001)
+    print(comments)
+
+    print("\n*Get a set of comments*")
+    comments = jira.issues_get_comments_by_id(10000, 10002)
```

Comment:
just a suggestion for the naming of function,
keeping in line with the single comment function naming convention,
could we keep multiple `comments` for the single `issue` named as,
`issue_get_comments_selective` rather than `issues_get_comments_by_id`
to help us understand that it is multiple selected comments for single issue.
Or some other better name? because i believe the filtering would always be based on the comment_id so keeping by_id mite be redundant here.

because it would always be comments drawn for the single issue and not multiple one if i understand correctly.

Reply:
Yeah, I'm happy to change it to this. My only reason for choosing: `issues_get_comments_by_id` was to be consistent with the Atlassian JIRA API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-comments/#api-rest-api-3-comment-list-post

`issues_get_comments_by_ids is still a bit weak`, so I'll consider selective unless I have an idea

I also chose: `issues_get_comments` rather than `get_issue_comments` to match some existing code

Reply:
that sounds fine then, looks good to me for merge, thanks!